platform/web/os_web.cpp

```diff
@@ -148,6 +148,10 @@ String OS_Web::get_unique_id() const {
 	ERR_FAIL_V_MSG("", "OS::get_unique_id() is not available on the Web platform.");
 }
 
+int OS_Web::get_default_thread_pool_size() const {
+	return godot_js_os_pthread_pool_size_get();
+}
```

Comment:
We don't even have to query JavaScript as we already know the answer.

```suggestion
int OS_Web::get_default_thread_pool_size() const {
#ifdef THREADS_ENABLED
	return godot_js_os_pthread_pool_size_get();
#else // Threads disabled.
	return 1;
#endif
}
```